### DISCUSSION TITLE
Gaps between windows in window management actions

### DISCUSSION THREAD

**Discussion by gjritter** [rxhanson/Multitouch-Community#227] [General]
Is it possible to use the various window management actions (centre half, first fourth, etc) with gaps between windows, similar to Rectangle? I like using centre half and first fourth/last fourth with 10 pixel gaps between windows, but I don't see an option to do this with Multitouch's settings.

**Comment by rxhanson** [comment]
You can add the gaps into Multitouch by using the following Terminal command, followed by restarting the app:

`defaults write com.brassmonkery.Multitouch gapSize -float 10`

I have plans to update Multitouch to include a lot more Rectangle UI preferences but haven't gotten to it yet. Let me know if you hit any issues with the gaps here.

**Comment by ronaldsuwandi** [comment]
@rxhanson does Multitouch also support the gap for screen edges like Rectangle 

https://github.com/rxhanson/Rectangle/blob/main/TerminalCommands.md#setting-gaps-at-the-screen-edges

**Reply by rxhanson** [reply]
It should! Replace `com.knollsoft.Rectangle` with `com.brassmonkery.Multitouch`.

```
defaults write com.brassmonkery.Multitouch screenEdgeGapTop -int 10
defaults write com.brassmonkery.Multitouch screenEdgeGapBottom -int 10
defaults write com.brassmonkery.Multitouch screenEdgeGapLeft -int 10
defaults write com.brassmonkery.Multitouch screenEdgeGapRight -int 10
```
Let me know if it doesn't work.

**Reply by ronaldsuwandi** [reply]
@rxhanson unfortunately it didn't work. For the time being I had to disable window management on Multitouch and use Rectangle until this is implemented in Multitouch 😆

**Reply by ronaldsuwandi** [reply]
@rxhanson Is there any plan to fix this? It's still doesn't work 🙏

**Reply by rxhanson** [reply]
Thanks for checking in. I'll take another look for this next release.

**Reply by rxhanson** [reply]
Sorry that I never got back to you on this. This is working in the current version of Multitouch. If for some reason it's not for you, then you can try appending .plist to the bundle id in the commands:

```
defaults write com.brassmonkery.Multitouch.plist screenEdgeGapTop -int 10
defaults write com.brassmonkery.Multitouch.plist screenEdgeGapBottom -int 10
defaults write com.brassmonkery.Multitouch.plist screenEdgeGapLeft -int 10
defaults write com.brassmonkery.Multitouch.plist screenEdgeGapRight -int 10
```

Then restart the app

**Reply by ronaldsuwandi** [reply]
@rxhanson weird, I'm on 1.34 and tried both 
```
defaults write com.brassmonkery.Multitouch screenEdgeGapBottom -int 28
defaults write com.brassmonkery.Multitouch.plist screenEdgeGapBottom -int 28
```

and restarted multitouch but it still doesn't respect that. See videocap below (for multitouch)

https://github.com/user-attachments/assets/dd0b43ec-540c-4d45-b07a-01d6ced2e3d3

When I use Left-Half on Finder the status bar is hidden behind sketchybar. In contrast comparing when I use Rectangle

https://github.com/user-attachments/assets/ee67584e-4d93-451f-a2e9-01373d437a29

Here it respects the screenEdgeGapBottom where I can see the finder status text

**Reply by rxhanson** [reply]
Figured it out! The next release with a fix will be out soon.
